runtime/queries/hare/indents.scm

```diff
@@ -0,0 +1,15 @@
+[
+ (struct_union_fields)
+ (expression_list)
+] @indent
+
+
+[
+  "{"
+  "}"
+] @branch
+
+[
+  (comment)
+] @ignore
```

Comment:
the helix indents mechanism doesn't do anything with `@branch` or `@ignore` captures: we could remove these wholesale or comment them out for now (in the future maybe a branch or ignore concept could be useful)